### PR TITLE
Updated readME for 7

### LINKED ISSUE #7: Update ReadMe

### PR DESCRIPTION
**Pull request by QUzair**
Closes #7

### CODE REVIEW

**Comment by hashdoge** [comment]
What the hell is this

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # StrongholdSiege
 
 This is a game.
+
+Is it a game though. It somehow didnt have haashims added text ;;;100000
```
Comment (salahusama) [inline README.md]: What is this?

Comment (hashdoge) [inline README.md]: ;;;;100000 <= ASS

Comment (QUzair) [reply README.md]: You wont understand, its an intellectual thing.

**Comment by hashdoge** [comment]
Looks good to go now buddy 👍

**Pull request merged**